rules/google_project_iam_audit_config_invalid_member.go

```diff
@@ -0,0 +1,78 @@
+package rules
+
+import (
+	"fmt"
+
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-google/project"
+)
+
+// GoogleProjectIamAuditConfigInvalidMemberRule checks whether member value is invalid
+type GoogleProjectIamAuditConfigInvalidMemberRule struct {
+	resourceType  string
+	blockName     string
+	attributeName string
+}
+
+func NewGoogleProjectIamAuditConfigInvalidMemberRule() *GoogleProjectIamAuditConfigInvalidMemberRule {
+	return &GoogleProjectIamAuditConfigInvalidMemberRule{
+		resourceType:  "google_project_iam_audit_config",
+		blockName:     "audit_log_config",
+		attributeName: "exempted_members",
+	}
+}
+
+// Name returns the rule name
+func (r *GoogleProjectIamAuditConfigInvalidMemberRule) Name() string {
+	return "google_project_iam_audit_config_invalid_member"
+}
+
+// Enabled returns whether the rule is enabled by default
+func (r *GoogleProjectIamAuditConfigInvalidMemberRule) Enabled() bool {
+	return true
+}
+
+// Severity returns the rule severity
+func (r *GoogleProjectIamAuditConfigInvalidMemberRule) Severity() string {
+	return tflint.ERROR
+}
+
+// Link returns the rule reference link
+func (r *GoogleProjectIamAuditConfigInvalidMemberRule) Link() string {
+	return project.ReferenceLink(r.Name())
```

Comment:
This returns a link to `docs/ rules/google_project_iam_audit_config_invalid_member.md`, so the corresponding documentation is needed. It may be better to create a dedicated page or combine these rules into a single document.